verilog/formatting/formatter_test.cc

```diff
@@ -1954,6 +1954,17 @@ static constexpr FormatterTestCase kFormatterTestCases[] = {
      "    localparam baz = 2  //comment\n"
      ") ();\n"
      "endmodule\n"},
+    {"module foo #("
+     "parameter  bar =1//comment\n"
+     ",localparam baz =2\n"
+     ") ();"
+     "endmodule",
+     "module foo #(\n"
+     "    parameter  bar = 1  //comment\n"
+     "    ,\n"
```

Comment:
Since the user intended to have the comma in front (I could imagine such being a style as unfortunately Verilog does not allow  for trailing commas in many places, so this style makes re-arranging things easier), is there a way for us to detect the situation and don't add an additional newline to the comma when we encounter it ? Because after the formatting, we didn't crash, but we also created a more vertical space-using output than before...

Reply:
Detection is possible, at least in unambiguous cases. Checking for line break between `,` and preceding token in source text should work.
Code that creates columns in align.cc needs adjustments to work with such cases.

I'll do required changes. In the meantime, https://github.com/chipsalliance/verible/pull/877 should be merged - changes from that PR will be useful.

Reply:
#877 is merged now. Could've been merged for a while as it had all approvals.